test/tools/libtesteth/TestOutputHelper.cpp

```diff
@@ -32,7 +32,7 @@ using namespace boost;
 
 Timer TestOutputHelper::m_timer;
 size_t TestOutputHelper::m_currTest = 0;
-size_t TestOutputHelper::m_maxTests = 0;
+size_t TestOutputHelper::m_maxTests = 1;
```

Comment:
Where was that?  Maybe you shouldn't use the class `TestOutputHelper` that always works with test sequence for the case when `m_maxTests` doesn't make sense